README.md

```diff
@@ -193,6 +193,23 @@ $ $EDITOR ~/.dcrd/dcrd.conf
 $ $EDITOR ~/.dcrwallet/dcrwallet.conf
 ```
 
+### Docker
+
+You can also run the wallet with docker:
+
+Volumes:
+
+- `/etc/dcrwallet` **mandatory** hold `config`, `rpc.cert ` and `rpc.key`
+- `/var/log/dcrwallet` is where the log files are stored
+- `/var/lib/dcrwallet` **important** hold the private keys, don't keep that volume ogn a temporary file system.
+
+Ports:
+
+- `9110` gRPC
+- `9111` Legacy RPC
+
+Any command passed to the container are `btcwallet` arguments, like `--help`.
```

Comment:
dcrwallet

Reply:
Fixed https://github.com/decred/dcrwallet/pull/999/commits/de81ed27e29527d6e736294674db967ae32f35a6

Sorry, I did some work on btcwallet before you forked it and the name stick in my brain :)